mappings/net/minecraft/util/profiling/jfr/JfrProfiler.mapping

```diff
@@ -0,0 +1,27 @@
+CLASS net/minecraft/class_6419 net/minecraft/util/profiling/jfr/JfrProfiler
+	FIELD field_34063 MINECRAFT Ljava/lang/String;
+	FIELD field_34064 WORLD_GENERATION Ljava/lang/String;
+	FIELD field_34065 TICKING Ljava/lang/String;
+	FIELD field_34066 NETWORK Ljava/lang/String;
+	FIELD field_34067 EVENTS Lcom/google/common/collect/ImmutableList;
+	FIELD field_34068 DATE_TIME_FORMAT Ljava/lang/String;
+	FIELD field_34069 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_34070 CONFIG_PATH Ljava/lang/String;
+	FIELD field_34071 currentRecording Ljdk/jfr/Recording;
+	METHOD method_37492 stop ()Lcom/mojang/datafixers/util/Either;
+	METHOD method_37493 start (Lnet/minecraft/class_6419$class_6420;)Z
+		ARG 0 instanceType
+	METHOD method_37494 (Lnet/minecraft/class_6419$class_6420;Ljava/lang/String;Ljdk/jfr/Recording;)V
+		ARG 2 recording
+	METHOD method_37495 start (Ljava/nio/file/Path;Lnet/minecraft/class_6419$class_6420;)Z
+		ARG 0 path
+		ARG 1 instanceType
+	METHOD method_37496 (Ljdk/jfr/Recording;)Z
+		ARG 0 recording
+	METHOD method_37498 isProfiling ()Z
+	CLASS 1
+		METHOD recordingStateChanged (Ljdk/jfr/Recording;)V
+			ARG 1 recording
+	CLASS class_6420 InstanceType
```

Comment:
fyi, InstanceType is `CLIENT`/`SERVER`. Its implementation is more like integrated/dedicated (this code will never run on MinecraftClient iirc) but `ServerType.CLIENT` is kinda weird.

Reply:
`net.fabricmc.api.EnvType` noises :laughing: